src/main/java/bootcamp/model/User.java

```diff
@@ -0,0 +1,62 @@
+package bootcamp.model;
+
+public class User {
```

Comment:
look up what `@JsonIgnoreProperties` is. Very useful annotation that we use a lot when dealing with POJOs.